infrastructure/helm-chart/BUILD

```diff
@@ -2,12 +2,11 @@ load("@rules_pkg//:pkg.bzl", "pkg_tar")
 load("@com_github_airyhq_bazel_tools//helm:helm.bzl", "helm_template_test")
 load("//tools/build:helm.bzl", "helm_push")
 
-
 filegroup(
     name = "files",
     srcs = glob(
         ["**/*"],
-        exclude = [ "BUILD" ],
+        exclude = ["BUILD"],
```

Comment:
was this intended?

Reply:
no, it was not intended! I just ran linting